src/modules/reservations/reservation-app.js

```diff
@@ -0,0 +1,65 @@
+import CarsApi from '../home/cars-api';
+
+const carApi = new CarsApi();
+
+const reservationsPopUp = document.createElement('div');
+const body = document.querySelector('body');
+
+export default function showReservationModal(e) {
+  const id = Number(e.target.parentNode.parentNode.id);
+
+  carApi.getDataPromise().then((data) => {
+    reservationsPopUp.classList.add('reservation-container');
+    reservationsPopUp.id = 'pop-up';
+    reservationsPopUp.hidden = true;
+    reservationsPopUp.style.display = 'block';
+
+    const car = data[id];
+
+    reservationsPopUp.innerHTML = `
+    <div class="reservation-content car-card bg-white flex flex-col gap-2 border-4 rounded-md border-violet-600" id="${id}">
+    <div class='flex justify-between'>
+     <img class="flex-1 bg-cover bg-contain w-78" src=${car.imgUrl}>
+     <span class='flex-1 close text-2xl'> X </span>
+    </div>
+    <h2 class='description'>${car.make} ${car.model} ${car.year}</h2>
+    <div>
+      <div class=' block flex justify-around'>
+       <p class='make '>Make: ${car.make}</p>
+       <p class='model'>Model: ${car.model}</p>
+      </div>
+      <div class='block flex justify-around' >
+       <p class='horsepower'>Horsepower: ${car.horsepower}</p>
+       <p class='price'>Price: ${car.price}</p>
+      </div>
+    </div>
```

Comment:
- [ ] [OPTIONAL] I believe it could be a nice addition to include the `Year` in the description of the car. Great job adding the others.